src/ostree/ot-builtin-refs.c

```diff
@@ -203,6 +214,11 @@ static gboolean do_ref (OstreeRepo *repo, const char *refspec_prefix, GCancellab
               const char *alias = g_hash_table_lookup (refs, ref);
               g_print ("%s -> %s\n", ref, alias);
```

Comment:
Hmm, maybe in `--revision` mode we should just print aliases like regular refs. Otherwise, it throws a wrench in the machine-friendliness of this output.

Reply:
Ahh gotcha, I mistakenly thought the `->` output could occur mixed with revision output.

> Would you like me to make it an error if they're combined?

That makes sense to me, though I'm not too worried about that corner case either.